scripts/generate_genotype_df/generate_genotype_df.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+"""Generate genotype dfs for the association analysis"""
+
+import hail as hl
+from analysis_runner import bucket_path, output_path
+
+TOB_WGS = bucket_path('mt/v7.mt/')
+
+def query():
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    mt = hl.read_matrix_table(TOB_WGS)
+    # filter out variants that didn't pass the VQSR filter
+    mt = mt.filter_rows(hl.is_missing(mt.filters))
```

Comment:
I thought Hail used an empty set for 0 applied filters? I've just tried this syntax in a notebook and all rows with no applied filters were removed.

This alternate syntax allows filters to be missing or a 0 length set, and will pass in both situations

```
matrix = matrix.filter_rows(
    hl.len(
        hl.or_else(
            matrix.filters, 
            hl.empty_set(hl.tstr)
        )
    ) == 0
)
```

Reply:
In the joint-calling pipeline we have empty sets for column-level filters, and NA for row-level filters. Good catch - should unify that. 

But agree that it's better to always handle both situations using the code you shared. Thanks Matt!

Reply:
that's really helpful, thanks @MattWellie!